sdk/polling_manager.ts

```diff
@@ -0,0 +1,31 @@
+import Flagsmith from '.';
+
+export class EnvironmentDataPollingManager {
+    private interval?: NodeJS.Timer;
+    private main: Flagsmith;
+    private refreshIntervalSeconds: number;
+
+    constructor(main: Flagsmith, refreshIntervalSeconds: number) {
+        this.main = main;
+        this.refreshIntervalSeconds = refreshIntervalSeconds;
+    }
+
+    start() {
+        const updateEnvironment = () => {
+            if (this.interval) clearInterval(this.interval);
+            this.interval = setInterval(async () => {
+                await this.main.updateEnvironment();
+            }, this.refreshIntervalSeconds * 1000);
+        };
+        // todo: this call should be awaited for getIdentityFlags/getEnvironmentFlags when enableLocalEvaluation is true
```

Comment:
Is there a reason this has been left as a todo?

Reply:
This was resolved in my pr, removing todo